API.md

```diff
@@ -50,6 +46,9 @@ attached to `getBlock` are errored with `Error('manual unwant: key')`.
 
 Cancel previously requested keys.
 
+### `putStream()`
+
+Returns a duplex `pull-stream` that emits an object `{key: Multihash}` for every written block when it was stored.
 
 ### `hasBlock(block, cb)`
```

Comment:
`has` is called when the block arrives? does it add the block? Shouldn't it just be put then?